### PR TITLE
Relax systemd units dependencies

### PR DESCRIPTION
**Pull request by ikapelyukhin**

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/66343/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 67f5533ca854cbe50c589346238b1973e623845c into 1fbe2e848f037bfa677f144a74461d3ae5318a66)

File: lib/rmt.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module RMT
-  VERSION ||= '2.4.5'.freeze
+  VERSION ||= '2.5.1'.freeze
```
[the remark this answers is not in the document]
Comment (ikapelyukhin) [reply lib/rmt.rb]: We didn't -- #502. I'll merge it in the correct order.

[the remark this answers is not in the document]
Comment (thutterer) [reply lib/rmt.rb]: Ah right, thanks!

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/67483/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 67f5533ca854cbe50c589346238b1973e623845c into 67627806e5f76714471a02433aca6766f51b98a1)

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/67484/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 67f5533ca854cbe50c589346238b1973e623845c into 26e66b5bbc9047d4603498e55cc885e9e40045fb)

**Pull request merged**